python/tvm/topi/cuda/conv2d_nhwc.py

```diff
@@ -54,6 +54,7 @@ def schedule_conv2d_nhwc_direct(cfg, s, Conv):
     cfg.define_knob("vthread_n", [1] if dynamic_batch else [1, 2])
     cfg.define_knob("vthread_c", [1, 2])
     cfg.define_knob("step", [16, 3, 32, 64])
+    cfg.define_knob("vectorize", [4, 2, 8, 16])
```

Comment:
Better to add a 1 for including no vectorize case.
```suggestion
    cfg.define_knob("vectorize", [1, 2, 4, 8, 16])
```